Improvements to documentation (#14)
KarlK90/yaemk-split-kb: docs/index.md

After placing your order you can proceed ordering the missing parts from [LCSC](https://lcsc.com/),
as they are both part of the same company,

> you [can save shipping costs](https://www.lcsc.com/faqs?id=140)

Reviewer: Great, thanks for the update!

They shine up through the PCB,
away from the surface the keyboard rests on.

> ![LED chain orientation](images/yaemk_led_chain.webp)

Reviewer: Thanks.

8. Solder RGB LEDs on the side facing away from you.
    Make sure to [correctly orient](#correct-orientation-of-rgb-leds) them,
    as they are not the same orientation all the time.
    Use the interactive BOM when in doubt. `D35 to D76`. Check that all LEDs
    are working by connecting the board to USB (the firmware from step 7 turns
    on all the LEDs). Note that as LEDs are [connected in

>     chain](#correct-orientation-of-rgb-leds)  - one dead LED leads
>     to many of them being off.

Reviewer: ```suggestion
    chain](#correct-orientation-of-rgb-leds)  - one dead LED leads
    to all subsequent leds being off.
```
